### DISCUSSION TITLE
Live Painting frequency

### DISCUSSION THREAD

**Discussion by Wangzhipeng777** [Acly/krita-ai-diffusion#471] [Q&A]
Hello, is there a frequency setting for real-time requests in Live Painting? I don't have the current requirement for extreme speed. For me, requesting every 30 seconds may be more in line with my needs. Are there any relevant parameters that can be adjusted?

**Comment by Acly** [comment]
There is no setting at the moment. But a new request will only be made when the previous one is complete, so if it takes longer there will also be fewer requests.
What is actually the issue? It is causing lag/stutter in the application?
